src/components/Date/Date.jsx

```diff
@@ -2,7 +2,6 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import _ from 'lodash';
-import moment from 'moment';
```

Comment:
I don't understand why our linter didn't complain about this being here and being unused, but when I removed all of the `moment` things from `validations.js` it was complaining. Maybe it is due to the different file extension.

Reply:
1. Our linter isn't running in CI, just in the commit hooks (should probably fix this sooner rather than later)
2. The linting commit hook only lints staged files, and this file hasn't been staged since it was added

Mind you, I don't know how it got here in the first place (from the old repo) without linting catching it. :man_shrugging: